tests/test_workflows/test_schemas.py

```diff
@@ -212,12 +212,12 @@ def test_valid_to_vcrs(get_topo_inputs_config, pipeline_topo, prefix, vcrs):
     [
         pytest.param(
             "wrong",
-            UserWarning,
+            ValueError,
```

Comment:
@adebardo With the update of this PR (we now properly catch `HTTPError` and other wrong string input that went undetected before), I couldn't easily adjust the error in this test.
I'm not sure I understood the logic: The `pyproj.CRS` EPSG error goes through normally, but the others don't and instead add something to logging? 
Now that error raising is more robust and clear directly in `vcrs`, should we simply raise the error every time? :thinking:

Reply:
Alright, I've resolved this using the `caplog` fixture to capture logging, and updating the test to check a logging error is raised instead.